.mergify.yml

```diff
@@ -0,0 +1,11 @@
+pull_request_rules:
```

Comment:
Maybe it would be better to exclude mergify and use the github native branch protection rules.
You would need to make a bracnch protection rule for master [like this](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule) and then activate auto merge [like this](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/incorporating-changes-from-a-pull-request/automatically-merging-a-pull-request)

Reply:
Very nice. I didn't knew that this is possible with GH. I only knew that from Azure DevOps. Very nice. Thanks for that!